package.json

```diff
@@ -36,7 +36,11 @@
 		"test-lint": "npm run prettier -- --check && eslint 'src/**/*.ts'",
 		"lint": "npm run prettier -- --write && eslint 'src/**/*.ts' --fix",
 		"prettier": "prettier './**/*.{js,ts,md,json,html,css}'",
-		"prepare": "npm run build"
+		"prepare": "npm run build",
+		"version": "npm test && git add .",
+		"release:patch": "npm version patch --message 'v%s' && npm publish && git push --follow-tags",
+		"release:minor": "npm version minor --message 'v%s' && npm publish && git push --follow-tags",
+		"release:major": "npm version major --message 'v%s' && npm publish && git push --follow-tags"
```

Comment:
We run one of the `release:*` scripts to publish to NPM. This builds and tests before publishing and finally pushing the updated files (including `dist/` and `src/Version.ts` files which are currently tracked in the repo) to GitHub along with the version tag.

Maybe we should remove publishing of `dist/` to GitHub, but that was out of scope of this PR.

Reply:
Note, I tested these scripts using `npm publish --dry-run` so that npm wouldn't actually publish anything.